asf_heat_pump_suitability/pipeline/run_scripts/run_add_features.py

```diff
@@ -139,28 +144,23 @@ def parse_arguments() -> argparse.Namespace:
     logging.info("Adding property density to EPC")
     epc_df = property_density.extend_df_with_property_density(epc_df)
 
+    # Add feature: property density Scotland
+    dz_density_df = property_density.generate_df_property_density_s()
+    epc_df = epc_df.join(dz_density_df, how="left", left_on="lsoa", right_on="DataZone")
```

Comment:
I was looking at the data in "s3://asf-heat-pump-suitability/outputs/2023Q4/20241105_2023_Q4_EPC_weighted_features.parquet" and there is no 'DataZone' column. I guess at some point you removed it (I couldn't see in this code why it wouldn't be saved out)? I don't think it matters, but thought I'd mention it in case it indicates an unintended issue.

Reply:
I think this is normal & expected behaviour. 'DataZone' is what we are using as the Scottish equivalent to 'LSOA'. Scottish DataZone codes appear in the `lsoa` column in EPC. When we do the join here, that's why we are joining on `lsoa` col from EPC and `DataZone` col from Scottish property density df. I think it's normal that the column to join on does not get duplicated in the resulting dataframe? So I don't think this is a problem but let me know if you still have concerns